herokuappTests/tests/test_booking.py

```diff
@@ -0,0 +1,25 @@
+
+from herokuappTests.base.base_class import Base
+
+
+class Booking(Base):
+    def test_create_booking(self):
+        booking_data = {
```

Comment:
Think about extracting this test data outside of the test if it is something that can be reused. 
For example, you can have default test data defined and then, for each test which needs some modification, just load that payload and change what you need.